src/widgets/biometrics/biometrics-chart.component.tsx

```diff
@@ -0,0 +1,136 @@
+  const [selectedBiometrics, setSelectedBiometrics] = React.useState<
+    biometricChartData
+  >({
+    title: `Weight (${weightUnit})`,
+    value: "weight",
+    groupName: "weight"
+  });
+  const [chartData, setChartData] = React.useState([]);
+
+  React.useEffect(() => {
+    const chartData = patientBiometrics.map(biometric => {
+      return {
+        group: selectedBiometrics.groupName,
+        key: dayjs(biometric.date).format("DD-MMM"),
+        value: biometric[selectedBiometrics.value]
+      };
+    });
+    setChartData(chartData);
+  }, [patientBiometrics, selectedBiometrics]);
+
+  const chartColors = {
+    weight: "#6929c4",
+    height: "#6929c4",
+    bmi: "#6929c4"
+  };
+  const chartOptions: LineChartOptions = {
+    axes: {
+      bottom: {
+        title: "Date",
+        mapsTo: "key",
+        scaleType: ScaleTypes.LABELS
+      },
+      left: {
+        mapsTo: "value",
+        title: selectedBiometrics.title,
+        scaleType: ScaleTypes.LINEAR,
+        includeZero: false
+      }
+    },
+    legend: {
+      enabled: false
+    },
+    color: {
+      scale: chartColors
+    }
+  };
+  return (
+    <div className={styles.biometricChartContainer}>
+      <div className={styles.biometricSignsArea} style={{ flex: 1 }}>
+        <label className={styles.biometricSign} htmlFor="radio-button-group">
+          Biometric Displayed
+        </label>
+        <RadioButtonGroup
+          defaultSelected="weight"
+          name="radio-button-group"
+          valueSelected="weight"
+          orientation="vertical"
+          labelPosition="right"
+        >
+          <RadioButton
+            id="weight"
+            labelText={`Weight (${weightUnit})`}
+            value="weight"
+            className={styles.biometricSignsRadioButton}
+            onClick={() =>
+              setSelectedBiometrics({
+                title: `weight (${weightUnit})`,
+                value: "weight",
+                groupName: "weight"
+              })
+            }
+          />
+          <RadioButton
+            id="height"
+            labelText={`Height (${heightUnit})`}
+            value="oxygenSaturation"
+            className={styles.biometricSignsRadioButton}
+            onClick={() =>
+              setSelectedBiometrics({
+                title: `Height (${heightUnit})`,
+                value: "height",
+                groupName: "height"
+              })
+            }
+          />
+          <RadioButton
+            id="bmi"
+            labelText={`BMI (${bmiUnit})`}
+            value="bmi"
+            className={styles.biometricSignsRadioButton}
+            onClick={() =>
+              setSelectedBiometrics({
+                title: `BMI (${bmiUnit})`,
+                value: "bmi",
+                groupName: "bmi"
+              })
+            }
+          />
+        </RadioButtonGroup>
+      </div>
+      <div className={styles.biometricSignsArea} style={{ flex: 4 }}>
```

Comment:
I haven't worked with flexbox that much, is there a reason why the flex styling is not included in the scss?

Reply:
No, Moved this styling to scss file